### PR TITLE
Correct hyperlinks in GitHub templates

### PR DESCRIPTION
**Pull request by TeslaCN**

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ on:
 jobs:
   build:
 
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```
Comment (Technoboy-) [inline .github/workflows/maven.yml:32]: I'm not sure about this. Shardingsphere uses macos.

**Comment by Technoboy-** [comment]
cc @terrymanu

File: .github/workflows/maven.yml (hunk 1 above)
Comment (TeslaCN) [reply .github/workflows/maven.yml:32]: I think Linux is more common in actual environment and it is a little faster than macOS in GitHub Action.

**Comment by TeslaCN** [comment]
I'm going to separate the commits in different PRs.

**Pull request closed without merging**